quiche/src/lib.rs

```diff
@@ -8570,6 +8577,9 @@ pub struct Stats {
 
     /// The total number of PATH_CHALLENGE frames that were received.
     pub path_challenge_rx_count: u64,
+
+    /// Total time during which bytes_in_flight > 0.
```

Comment:
nit: maybe that's obvious but I would specify in the doc that the "bytes in flight" are bytes we are sending. So I guess "unacked sent bytes" is another way to describe it.

Reply:
Attempted to clarify.